ssl/ssl_lib.c

```diff
@@ -694,6 +694,26 @@ int SSL_CTX_set_ssl_version(SSL_CTX *ctx, const SSL_METHOD *meth)
 }
 #endif
 
+SSL *SSL_new_ex(SSL_CTX *ctx, const char* propq)
```

Comment:
Is this better than introducing another call to set this?
That's how a lot of the other SSL settings that also exist in the SSL_CTX are implemented.